### PR TITLE
return Change enum if byte was flipped

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Was needed for hypercore. Returns whether a byte was changed.

### CODE REVIEW

**Comment by yoshuawuyts** [comment]
Or alternatively: we should create an enum with `Changed`, `Unchanged`. That might be slightly neater tbh.

**Comment by yoshuawuyts** [comment]
yeah, this feels a lot better! :D

**Comment by yoshuawuyts** [comment]
tests failing b/c clippy doesn't compile. Ah well, guess that's what happens on nightlies

**Review by ralphtheninja — APPROVED** [review]

File: src/lib.rs [hunk 1]
```diff
@@ -17,6 +17,7 @@ extern crate memory_pager;
 use memory_pager::Pager;
 
 /// Bitfield instance.
+#[derive(Debug)]
```
Comment (ralphtheninja) [inline src/lib.rs:20]: What does this mean?

Comment (pfrazee) [reply src/lib.rs:20]: It automatically adds a stringified form of the value for if you ever log a `Bitfield` to the console

Comment (yoshuawuyts) [reply src/lib.rs:20]: Yep! - also good to know there is also a `Display` trait you can implement, which acts slightly differently. The main difference being that you must define the formatting yourself. 

- `println!("{:?}", my_value);` is for the `Debug` trait
- `println!("{}", my_value);` is for the `Display` trait

**Comment by yoshuawuyts** [comment]
follow-up commit in Hypercore here: https://github.com/datrs/hypercore/commit/2fdf77c48f4e08b6de7da97f6ae979085f67567b

**Pull request merged**